lib/src/tooltip_action.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:flutter/material.dart';
+
+import 'showcase_context_provider.dart';
+import 'showcase_widget.dart';
+
+class ToolTipAction extends StatelessWidget {
+  const ToolTipAction.defaultAction({
+    Key? key,
+    this.actionOne,
+    this.actionTwo,
+    this.actionPadding = const EdgeInsets.only(top: 5),
+    this.alignment = WrapAlignment.spaceBetween,
+    this.crossAxisAlignment = WrapCrossAlignment.center,
+  })  : actions = null,
+        super(key: key);
+
+  const ToolTipAction.customAction({
+    Key? key,
+    this.actions,
+    this.actionPadding = const EdgeInsets.only(top: 5),
+    this.alignment = WrapAlignment.spaceBetween,
+    this.crossAxisAlignment = WrapCrossAlignment.center,
+  })  : actionOne = null,
+        actionTwo = null,
+        super(key: key);
+
+  /// Define configuration of first action
+  final ActionWidgetConfig? actionOne;
+
+  /// Define configuration of second action
+  final ActionWidgetConfig? actionTwo;
+
+  /// Define padding of action widget
+  final EdgeInsets? actionPadding;
+
+  /// Define configuration of list of action
+  final List<ActionWidgetConfig>? actions;
+
+  /// Define alignment of actions
+  final WrapAlignment alignment;
+
+  /// Define crossAxisAlignment of actions
+  final WrapCrossAlignment crossAxisAlignment;
+
+  @override
+  Widget build(BuildContext context) {
+    final showcaseContext = ShowcaseContextProvider.of(context)?.context;
+    return Padding(
+      padding: actionPadding ?? EdgeInsets.zero,
+      child: Wrap(
+        alignment: alignment,
+        crossAxisAlignment: crossAxisAlignment,
+        children: actions != null
+            ? [for (final action in actions!) getActionWidget(action)]
+            : [
+                getActionWidget(
+                  actionOne ??
+                      ActionWidgetConfig(
+                        actionTitle: 'Prev',
+                        onActionTap: () {
+                          if (showcaseContext == null) return;
+                          ShowCaseWidget.of(showcaseContext).previous();
+                        },
+                      ),
+                ),
+                getActionWidget(
+                  actionTwo ??
```

Comment:
What is user doesn't want to provide 2nd action. I think Instead of making user to pass a sizedbox, we should only show this if they have provided the config.